src/applications/personalization/dashboard-2/tests/e2e/dashboard-loa3.cypress.spec.js

```diff
@@ -0,0 +1,71 @@
+import disableFTUXModals from '~/platform/user/tests/disableFTUXModals';
+import { loa3User } from '@@profile/tests/fixtures/users/user.js';
+import serviceHistory from '@@profile/tests/fixtures/service-history-success.json';
+import fullName from '@@profile/tests/fixtures/full-name-success.json';
+import disabilityRating from '@@profile/tests/fixtures/disability-rating-success.json';
+
+import manifest from 'applications/personalization/dashboard/manifest.json';
+
+import { mockFeatureToggles } from './helpers';
+
+/**
+ *
+ * @param {boolean} mobile - test on a mobile viewport or not
+ *
+ * This helper:
+ * - loads the my VA Dashboard,
+ *   checks that focus is managed correctly, and performs an aXe scan
+ */
+function loa3DashboardTest(mobile) {
+  mockFeatureToggles();
+  cy.visit(manifest.rootUrl);
+
+  if (mobile) {
+    cy.viewport('iphone-4');
+  }
+
+  // should show a loading indicator
+  cy.findByRole('progressbar').should('exist');
+  cy.findByText(/loading your information/i).should('exist');
+
+  // and then the loading indicator should be removed
+  cy.findByRole('progressbar').should('not.exist');
+  cy.findByText(/loading your information/i).should('not.exist');
+
+  // focus should be on the h1
+  cy.focused()
+    .should('have.attr', 'id', 'dashboard-title')
+    .contains('My VA')
+    .and('have.prop', 'tagName')
+    .should('equal', 'H1');
+
+  // name tag exists with the right data
+  cy.findByTestId('name-tag').should('exist');
+  cy.findByText('Wesley Watson Ford').should('exist');
+  cy.findByText('United States Air Force').should('exist');
+  cy.findByText('90% Service connected').should('exist');
```

Comment:
These three `findByText`s should likely be smarter. It might be better if, instead of hardcoding the strings we are looking for, we used the contents of the mock API responses. So it would effectively read like "the name contained in `loa3User` should exist in the DOM"

Reply:
Yes, I very much agree with that. We don't want flaky tests when those user objects change for whatever reason 💯